src/applications/fry-dea/components/ConfirmationResponses.jsx

```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
   response: PropTypes.object || PropTypes.bool,
 };
 
+export function Approved(name) {
+  return (
+    <>
+      <div>
+        <va-alert
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 id="track-your-status-on-mobile" slot="headline">
+            Congratulations, you’ve been approved for Survivors’ and Dependents’
+            Educational Assistance
+          </h2>
+          <div>
+            <p className="vads-u-margin-y--0">
+              We reviewed your application and have determined that you’re
+              entitled to educational benefits under Survivors’ and Dependents’
+              Educational Assistance (DEA, Chapter 35). Your decision letter is
+              now available. A physical copy will also be mailed to your mailing
+              address.
+            </p>
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.va.gov"
+              text="Download your decision letter"
```

Comment:
According to the mockup, there should be a period at the end of this sentence (outside of the link, though).

Reply:
resolve